src/NuGet.Clients/NuGet.CommandLine/Commands/OptionAttribute.cs

```diff
@@ -8,6 +8,7 @@ public sealed class OptionAttribute : Attribute
     {
         private string _description;
 
+        public bool Hidden { get; set; }
```

Comment:
It's helpful to have bool properties start with "Is", "Can", "Has", etc. when it adds value and I believe this is a case where `IsHidden` is more clear than `Hidden`. See [property naming guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members?redirectedfrom=MSDN#names-of-properties)

```suggestion
        public bool IsHidden { get; set; }
```